lms/djangoapps/certificates/tests/test_generation_handler.py

```diff
@@ -32,18 +32,13 @@
     GeneratedCertificateFactory,
     CertificateInvalidationFactory
 )
-from openedx.core.djangoapps.certificates.config import waffle
+from openedx.core.djangoapps.certificates.config.waffle import AUTO_CERTIFICATE_GENERATION
 
 log = logging.getLogger(__name__)
 
 ID_VERIFIED_METHOD = 'lms.djangoapps.verify_student.services.IDVerificationService.user_is_verified'
-AUTO_GENERATION_NAMESPACE = waffle.WAFFLE_NAMESPACE
-AUTO_GENERATION_NAME = waffle.AUTO_CERTIFICATE_GENERATION
-AUTO_GENERATION_SWITCH_NAME = '{}.{}'.format(AUTO_GENERATION_NAMESPACE, AUTO_GENERATION_NAME)
-AUTO_GENERATION_SWITCH = LegacyWaffleSwitch(AUTO_GENERATION_NAMESPACE, AUTO_GENERATION_NAME)
 
-
-@override_switch(AUTO_GENERATION_SWITCH_NAME, active=True)
+@override_switch(AUTO_CERTIFICATE_GENERATION, active=True)
```

Comment:
Instead of the waffle's `override_switch` decorator, please use edx_toggles `override_waffle_switch`. Same thing below in the test_tasks module.

Reply:
@regisb I've done the changes.